package.json

```diff
@@ -48,10 +42,6 @@
     "@types/semver": "7.3.13",
     "chalk": "4.1.2",
     "concurrently": "7.6.0",
-    "cypress": "12.8.0",
```

Comment:
Put the comment here to have the nice comment PR review management 😁

I can see one `data-cy` left, I think it was because of bad merge timing management. ` data-cy="push-changes"` in `packages/slice-machine/src/features/changes/PushChangesButton.tsx`. I don't even think it's used so you could remove it.

Reply:
Removed!